python/xorbits/deploy/docker/Dockerfile.base

```diff
@@ -23,13 +23,15 @@ ARG PYTHON_VERSION=3.9
 RUN if [ "$PYTHON_VERSION" == "3.11" ] ; then \
     /opt/conda/bin/conda install --freeze-installed -c conda-forge \
     python=3.11 numpy\>=1.14.0 pandas\>=1.5.0 cython\>=0.29.33 scipy\>=1.9.2 \
-    mkl numexpr psutil scikit-learn sqlalchemy tornado lz4 ; fi
+    mkl numexpr jax jaxlib psutil scikit-learn sqlalchemy tornado lz4 ; fi
```

Comment:
Installing ``jax`` and ``jaxlib`` on ``final`` docker layer would be enough. Is there any special reason to install it again under python 3.11 ?

Reply:
Hi, I just followed the installation steps for numexpr, which appears in both places. I am not sure if there is a special purpose to it, but I suppose I will remove this part then? Thanks.

Reply:
> Hi, I just followed the installation steps for numexpr, which appears in both places. I am not sure if there is a special purpose to it. So maybe I will remove this part then? Thanks.

Here's the thing, python 3.11 is a relatively new version of python, and the dependencies we need like pandas and numpy have unique requirements for this python version, so the 3.11 layer installs these dependencies separately while specifying the ``--freeze-installed`` option. ``jax`` and ``jaxlib`` should just install normally for each python version. And ``jax`` and ``jaxlib`` should be installed via the ``conda-forge`` channel according to ``conda search`` result.

Therefore, add ``jax`` and ``jaxlib`` to ``conda-forge`` installation on ``final`` layer (docker file line 39-42) would be good.

Reply:
I see, thanks, i have removed the install